.github/workflows/release.yml

```diff
@@ -8,6 +8,8 @@ jobs:
   make-release:
     name: Make and publish spec release
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
```suggestion
      contents: write # required to publish (tag) a release 
```

Reply:
Good catch! Did GitHub not inform us that our last builds failed?

Reply:
our most recent builds exited early because the spec itself had not changed (dependency updates and affiliation update).